include/swift/AST/DiagnosticsSema.def

```diff
@@ -7977,7 +7977,13 @@ ERROR(lifetime_dependence_cannot_infer_ambiguous_candidate, none,
        ())
 ERROR(lifetime_dependence_immortal_conflict_name, none,
       "conflict between the parameter name and immortal keyword", ())
-
+ERROR(lifetime_dependence_function_type, none,
+      "Lifetime dependencies on function types are not yet supported",
```

Comment:
Please use all lower-case spelling and I think we are trying to avoid using “yet” in this sort of diagnostic in other places.

Reply:
Done!